hedera-node/src/main/java/com/hedera/services/txns/file/FileAppendTransitionLogic.java

```diff
@@ -95,8 +95,9 @@ public void doStateTransition() {
 				return;
 			}
 
-			var result = hfs.append(target, data);
-			txnCtx.setStatus(result.outcome());
+			final var result = hfs.append(target, data);
+			final var status = result.outcome();
```

Comment:
We dont need to mark entity changed here ? `sigImpactHistorian.markEntityChanged`

Reply:
In the future, we may want to generalize the `sigImpactHistorian` to a `changeHistorian`; so that **any** change to an entity causes it to be marked as changed.

But right now, **only** changes that can impact the result of `SigRequirements.keyForPayer()` and `SigRequirements.keysForOtherPayers()` cause anything to be marked as changed.

Reply:
Ah I see. Got it. Thank you!